src/njoy21/input/DTFR/Card3a.hpp

```diff
@@ -0,0 +1,30 @@
+class Card3a {
+public:
+#include "njoy21/input/DTFR/Card3a/Mti.hpp"
+#include "njoy21/input/DTFR/Card3a/Mtc.hpp"
+#include "njoy21/input/DTFR/Card3a/Nlc.hpp"
+
+  Argument< Mti > mti;
+  Argument< Mtc > mtc;
+  Argument< Nlc > nlc;
+  
+  template< typename Char >
+  Card3a( iRecordStream< Char >& is )
+    try:
+      mti ( argument::extract< Mti >( is ) ),
+      mtc ( argument::extract< Mtc >( is ) ),
+      nlc ( argument::extract< Nlc >( is ) )
+      {
+        Card::clear(is);
+      }
+    catch( std::exception& e ){
+      Log::info("Trouble while validating Card3a");
+      throw e;
+    }
+
+    friend void swap( Card3a& left, Card3a& right ){
```

Comment:
A large error message comes up when trying to do the Card3a test, but I'm having trouble seeing where swap is being called from